index.mjs

```diff
@@ -140,6 +140,7 @@ fs.readdir('./', (err, files) => {
           icon.outline,
           icon.brand,
           icon.categories,
+          [],
```

Comment:
Note: Always empty array since the aliases from  `static/aliases.json` has no search terms.